desktop/apps/artist/components/current_venice_feature/index.coffee

```diff
@@ -0,0 +1,42 @@
+_ = require 'underscore'
+Q = require 'bluebird-q'
+request = require 'superagent'
+cache = require '../../../../lib/cache.coffee'
+
+cacheKey = "venice-2017"
+
+fetchVeniceArtists = ->
+  Q.promise (resolve, reject) ->
+    cache.get cacheKey, (err, cachedData) ->
+      return reject(err) if err
+      return resolve(JSON.parse(cachedData)) if cachedData
+
+      request
+        .get "http://files.artsy.net/force/venice_2017_artists.json"
+        .end (err, res) =>
+          if res?.ok
+            try
+              data = JSON.parse res.text
+              cache.set "venice-2017", res.text
+              resolve(data)
+            catch e
+              reject(e)
+          else
+            reject(res?.error or err.response)
+
+module.exports = (artist, fetch = fetchVeniceArtists) ->
+  fetch()
+    .then (veniceArtists) ->
+      for veniceArtist in veniceArtists
+        if veniceArtist.slug == artist.id
+          return {
+            type: 'eoy',
+            href: '/venice-biennale',
+            imageUrl: 'http://files.artsy.net/images/venice_2017.jpg',
```

Comment:
wonder if we'd get that 'mixed content' warning on browsers b/c of the SSL comment ^